packages/nocms-plugin-json-resources/src/createJsonResourceProvider.js

```diff
@@ -3,7 +3,19 @@ import path from 'path';
 const jsonResourceGlobPattern = '**/!(_)*.json';
 const jsonResourceType = 'json';
 
-export default function createJsonResourceProvider ({findFiles, readFile, watchFiles, writeFile}) {
+function canCompileJsonResource (jsonResource) {
+	return jsonResource.type === jsonResourceType;
+}
+
+/**
+ * createJsonResourceId prefixes a path with `/` and converts its path separator to `/`
+ * @param {String} jsonFile a path to a json file
+ */
+export function createJsonResourceId (jsonFile) {
+	return '/' + jsonFile.split(path.sep).join('/');
```

Comment:
String literals :-D

Reply:
I am not sure I understand this one. I would not expect an `Id` to have slashes in it?

Reply:
Id does get prefixed with a slash according to the working directory of nocms as far as I'm aware. It's not an absolute file path, but it is from the root of where nocms is run